configs/src/deployments/json/modeMainnet.json

```diff
@@ -0,0 +1,184 @@
+{
+  "v1.3.0": {
+    "AddresslistVotingSetup": {
+      "address": "0x7a62da7B56fB3bfCdF70E900787010Bc4c9Ca42e",
+      "blockNumber": 13627009,
+      "deploymentTx": "0xc3e18ea335f8f00f6b192c228ca3ac9508b5837636a20e1f491f3f5b35d19494"
+    },
+    "AddresslistVotingSetupImplementation": {
+      "address": "0x2C9c5e8F559DBBEc962f7CCd295DBc4183cd2168",
+      "blockNumber": 13627009,
+      "deploymentTx": "0xc3e18ea335f8f00f6b192c228ca3ac9508b5837636a20e1f491f3f5b35d19494"
+    },
+    "AdminSetup": {
+      "address": "0xE978942c691e43f65c1B7c7F8f1dc8cDF061B13f",
+      "blockNumber": 13627027,
+      "deploymentTx": "0xe5427803530b4ad9c0649fa11096887c1a8b24bc0ff70268758c0d23e1b69ebc"
+    },
+    "AdminSetupImplementation": {
+      "address": "0xe015E365572c1d778be7E3ACf16f6a0C06dE86f0",
+      "blockNumber": 13627027,
+      "deploymentTx": "0xe5427803530b4ad9c0649fa11096887c1a8b24bc0ff70268758c0d23e1b69ebc"
+    },
+    "DAOBase": {
+      "address": "0x741061B01d3A95a7734B594f1884b6c30E0fDA96",
```

Comment:
according to daoFactory https://modescan.io/address/0x08633901DdF9cD8e2DC3a073594d0A7DaD6f3f57/contract/34443/code
daoBase should be [0xbf8b2c4062E1d1060c332bF3305EF03971C90859](https://modescan.io/address/0xbf8b2c4062E1d1060c332bF3305EF03971C90859)